build_tools/github/test_wheels.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ "$RUNNER_OS" != "Linux" ]]; then
+    # The Linux test environment is run in a Docker container and
+    # it is not possible to copy the test configuration file (yet)
+    cp $CONFTEST_PATH $CONFTEST_NAME
```

Comment:
Maybe using single quote escaping is necessary for windows filepath:

```suggestion
    cp '$CONFTEST_PATH' $CONFTEST_NAME
```

see: https://stackoverflow.com/questions/7962485/escaping-backslash-in-windows-paths-passed-to-unix-programs

Reply:
Moving `CONFTEST_PATH` and `CONFTEST_NAME` outside of `CIBW_ENVIRONMENT` is working too (they are not really needed for `cibuildwheel`).

I think that this last commit will work properly 😉.

Reply:
> Moving CONFTEST_PATH and CONFTEST_NAME outside of CIBW_ENVIRONMENT is working too (they are not really needed for cibuildwheel).

+1

> Nevertheless, I could apply your proposed solution if you think that it would be better.

If it's needed yes. I let you choose.